scripts/sort_camera_trap_images.py

```diff
@@ -21,30 +21,45 @@
 import shutil
 import datetime
 import json_database
+import tools
 
-def sort_camera_trap_images( unsorted_dir ):
+def sort_camera_trap_images( unsorted_dir, individual ):
 
   json_name = os.path.basename(os.path.dirname(unsorted_dir)) + ".json"
   snapcat_database_dir = os.path.join( unsorted_dir, json_name )
   snapcat_json = json_database.JSONDatabase( snapcat_database_dir )
   
   burst.create_bursts( snapcat_json, unsorted_dir )
-  #segmentation.segment_images( snapcat_json )
+  segmentation.segment_images( snapcat_json )
```

Comment:
we may want to allow some argument if we want to sort using the classifier or sort using the user scripts.